contrib/images/simd-env/Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM golang:1.17-alpine AS build
 RUN apk add build-base git linux-headers
 WORKDIR /work
 COPY go.mod go.sum /work/
-COPY db/go.mod db/go.sum /work/db/
```

Comment:
This was deleted as the db/ folder does not exist in cosmos-sdk `v0.44.5` and thus causes a `make localnet-start` to fail as well as failing tests.